### ISSUE TITLE
Attribute Error in Notebook 

### ISSUE THREAD

**Issue by ckucewicz** [learn-co-curriculum/dsc-hypothesis-testing-lab#11]
## Link to Canvas

https://learning.flatironschool.com/courses/7075/quizzes/50685?module_item_id=602450

## Issue Subtype

- [x ] Master branch code
- [ ] Solution branch code
- [ x] Code tests
- [ ] Layout/rendering issue
- [ ] Instructions unclear
- [ ] Other (explain below)

### Source
<# Run this cell without changes
import statsmodels.api as sm
from statsmodels.formula.api import ols

formula = 'PHYSHLTH ~ C(_STATE)'
lm = ols(formula, df).fit()
sm.stats.anova_lm(lm)>
```

```

### Concern

When I run the code without changes as specified, I get an AttributeError: module 'numpy' has no attribute 'MachAr'

## What OS Are You Using?

- [X ] OS X
- [ ] Windows
- [ ] WSL
- [ ] Linux
- [ ] Saturn Cloud from Canvas

**Comment by bpurdy-ds** [comment]
Thank you for raising this issues. We'll take a look.

**Comment by bpurdy-ds** [comment]
Hello again. I just ran the code and it is running fine.
I googled the error and many of the proposed solutions were of this sort:
https://stackoverflow.com/questions/75371176/module-numpy-has-no-attribute-machar
I would reach out to your instructor if you are still having issues after looking at what others have done to try to fix this issue.

**Comment by ckucewicz** [comment]
Thank you for providing this insight! I’ll reach out to my instructor if I am having any more difficulties.Best regards,Chris Kucewicz Sent from my iPhoneOn May 7, 2024, at 10:04 AM, Brendan Patrick Purdy ***@***.***> wrote:﻿
Hello again. I just ran the code and it is running fine.
I googled the error and many of the proposed solutions were of this sort:
https://stackoverflow.com/questions/75371176/module-numpy-has-no-attribute-machar
I would reach out to your instructor if you are still having issues after looking at what others have done to try to fix this issue.
